server/test/handlers/cancelSpecificDay.test.js

```diff
@@ -0,0 +1,23 @@
+const appointmentHandler = require('../../models/handlers/appointment');
+
+describe('testing cancellation of worker appointments on specific day', () => {
+    beforeEach(() => {
+        jest.resetModules();
+    });
+
+    test('initial setup of worker cancel specific day functionality', async() => {
+        // Arrange
+        const workerId = 8000000;
+        const startTime = '2020-12-20 08:00:00';
+        const endTime = '2020-12-20 10:30:00';
```

Comment:
See previous comment

Reply:
Hi Peter, I have responded to your previous comment. Please let me know if anything is unclear or if you would still prefer the split.

Reply:
All clear thank you Amy!